quidem-ut/pom.xml

```diff
@@ -488,6 +492,11 @@
             <version>0.9.3</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.kgyrtkirk.datasets</groupId>
+            <artifactId>kttm-nested</artifactId>
+            <version>0.1</version>
+        </dependency>
```

Comment:
this seems like an odd way to get this test file...

Reply:
could be seen as odd - however it has quite a few benefits:
* offloads the download/caching to maven
* utilizes java/jar to keep the file compressed ; but still accessible in an uncompressed form

do you have something else in mind?